src/renderer/store/handlers/identity.ts

```diff
@@ -315,6 +324,12 @@ export const createIdentity = ({ name, fromMigrationFile }) => async () => {
       signerPubKey = keys.signerPubKey
     }
 
+    const certString = await readRootCertFromFile()
```

Comment:
this cannot work in renderer as this reads from files on disk.

Maybe do yourself a favour and move root_ca and root_key as keys into `src/shared/static.js` and lets drop disk operations altogether?